packages/genesys-spark-components/src/test/playwrightTestUtils.ts

```diff
@@ -72,9 +72,9 @@ export async function snap(
 
   if (await page.locator('gux-tooltip').isVisible()) {
     expect(await page.screenshot({ animations })).toMatchSnapshot();
-  } else if (element && (await page.locator(element).isVisible())) {
+  } else if (element && (await page.locator(element).first().isVisible())) {
```

Comment:
I added this because I wanted to retrieve the first `gux-rating` element in the test not the one that is internal when the `compact` version is active. I do not see any downside to this.